Src/library/net/Clib/network.c

```diff
@@ -29,12 +29,15 @@ indexing
 #define FD_SETSIZE 256
 #include <winsock2.h>
 #include <ws2tcpip.h>
+#include <Mswsock.h>
```

Comment:
Isn't it too much if a single send over network is accompanied with DLL load/unload? I'd better keep the function address somewhere and do not unload the DLL unless the Eiffel object to do the transmissions is released. Then the DLL can be unloaded.